dts/arm/st/u5/stm32u5.dtsi

```diff
@@ -399,6 +399,20 @@
 			status = "disabled";
 			label = "UCPD_1";
 		};
+
+		gpdma1: dma@40020000 {
+			compatible = "st,stm32-dma-v3";
+			#dma-cells = <3>;
+			reg = <0x40020000 0x400>;
+			interrupts = <29 0 30 0 31 0 32 0 33 0 34 0 35 0 36 0
+					80 0 81 0 82 0 83 0 84 0 85 0 86 0 87 0>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB1 0x1>;
+			dma-channels = <16>;
+			dma-requests = <114>;
+			dma-offset = <0>;
+			status = "disabled";
+			label = "DMA_1";
```

Comment:
Should it be GPDMA1 ?

Reply:
DMA is well known, GPDMA could be too specific,  and moreover it's _just_ a label